.github/workflows/pr_push.yml

```diff
@@ -71,7 +71,7 @@ jobs:
       - name: Install apt packages
         run: |
           sudo apt-get update
-          sudo apt-get install -y cmake clang-format-15
+          sudo apt-get install -y cmake clang-format-15 libnuma-dev
```

Comment:
I don't think it's required in coding style - perhaps just disable building of the tests

Reply:
Done.